src/main/java/org/gridsuite/modification/server/entities/ElementaryModificationEntity.java

```diff
@@ -0,0 +1,56 @@
+/*
+  Copyright (c) 2021, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.entities;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.gridsuite.modification.server.ModificationType;
+import org.gridsuite.modification.server.dto.ElementaryModificationInfos;
+
+import javax.persistence.*;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.Set;
+
+/**
+ * @author Slimane Amar <slimane.amar at rte-france.com>
+ */
+@NoArgsConstructor
+@Getter
+@Entity
+@Table(name = "elementaryModification")
+public class ElementaryModificationEntity extends AbstractModificationEntity {
+    @Column(name = "equipmentId")
+    private String equipmentId;
+
+    @Transient
+    private Set<String> substationIds = Set.of();
+
+    @OneToOne(fetch = FetchType.EAGER, optional = false, orphanRemoval = true, cascade = {CascadeType.ALL})
```

Comment:
we try to never use EAGER, can it be avoided ?

Reply:
Yes
I haven't watched yet the latest study-server

Reply:
Done